Sources/SnappTheming/Theme/SegmentControls/SnappThemingSegmentControlStyleConfiguration.swift

```diff
@@ -6,6 +6,7 @@
 //
 
 import Foundation
+import SwiftUI
```

Comment:
```suggestion
import SwiftUI
```
Probably not need here as well (but please double check.